### ISSUE TITLE
JS Gets JSON

### ISSUE THREAD

**Issue by wylie** [wylie/ifeelblank.com#12]
Right now we save the feelings to an html file...  Make the JS save to a JSON file using PHP (which relies on #1). This will need to be completed after #3 is also completed.

**Comment by wylie** [comment]
We are now GETting the JSON data from the new JSON file. No saving to the JSON file yet, but that will come shortly... I suppose a new issue for that is in order.
